jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -48,7 +48,11 @@ cp "${DATA}/soca_gridspec.nc" "${COMOUT}/${CDUMP}.t${bcyc}z.ocngrid.nc"
 
 # Copy logs
 mkdir -p "${COMOUT}/logs"
-cp "${DATA}/*.out" "${COMOUT}/logs"
+cp "${DATA}"/*.out "${COMOUT}/logs"
```

Comment:
Note that this space does not and will not exist in ops.

Reply:
@aerorahul , what would be the appropriate place to save the configuration/yamls of the cycle?

Reply:
That depends on the purpose of this data.  Is this data used in subsequent cycles?  That is the purpose of COM.
Is this for validation?